__tests__/app.test.js

```diff
@@ -41,4 +41,45 @@ describe( "/api/", ()=> {
             expect(body.allEndPoints).toEqual(endpoints)
             })
         })
-    })
+})
+
+describe( "/api/articles/:article_id", ()=> {
+    it("GET 200 sends the selected article by given article_id", () => {
+        return request(app)
+        .get("/api/articles/1")
```

Comment:
You are requesting article 1, but expecting literally anything. How does this test give me and you confidence that your query is correct?